spec/features/contestant_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.feature "Contestants", type: :feature do
+  it "can see a contestants name and a list of projects they have worked on" do
```

Comment:
This test could be a bit more robust if you had another contestant that was part of multiple projects, so you could verify that your code was listing all the projects they were in rather than just one